cfme/tests/containers/test_one_image_analysis.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import flash, toolbar as tb, tabstrip as tabs
+from utils.wait import wait_for, TimedOutError
+from cfme.configure import tasks
+
+# Polarion test case CMP-10064
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+def test_image_analysis_finished():
+    sel.force_navigate('containers_images')
+    checkbox = ".//input[@class='list-grid-checkbox']"
+    sel.check(checkbox)
+    tb.select(
+        'Configuration',
+        'Perform SmartState Analysis',
+        invokes_alert=True)
+    sel.handle_alert()
+    flash.assert_message_contain('Analysis successfully initiated')
+    success_alert = ".//div[@class='alert alert-success']"
+
+    if not success_alert:
+        wait_for(lambda: sel.is_displayed(success_alert), num_sec=10, delay=5)
+
+    is_task_finished()
+
+
+def is_task_finished():
```

Comment:
I guess you could use the `all_vm_analysis_tasks` function inside to look up the task. It is located in the cfme/configure/tasks.py, line 156. It returns list of dicts, you can try that out in ipython.